docs/getting-started.md

```diff
@@ -176,22 +176,29 @@ Follow the [installation instructions for your Linux distribution](https://nodej
 
 <block class='native windows android' />
 
-<h3>Node, Python2, JDK</h3>
+<h3>Node, Python, JDK</h3>
 
-We recommend installing Node and Python2 via [Chocolatey](https://chocolatey.org), a popular package manager for Windows.
+We recommend installing Node and Python via [Chocolatey](https://chocolatey.org), a popular package manager for Windows.
 
-React Native also requires a recent version of the [Java SE Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html), as well as Python 2. Both can be installed using Chocolatey.
+React Native also requires a recent version of the [Java SE Development Kit (JDK)](https://openjdk.java.net/projects/jdk/14/), as well as Python. Both can be installed using Chocolatey.
 
 Open an Administrator Command Prompt (right click Command Prompt and select "Run as Administrator"), then run the following command:
 
 ```powershell
-choco install -y nodejs.install python2 jdk8
+choco install -y nodejs.install python openjdk 
 ```
 
 If you have already installed Node on your system, make sure it is Node 8.3 or newer. If you already have a JDK on your system, make sure it is version 8 or newer.
 
 > You can find additional installation options on [Node's Downloads page](https://nodejs.org/en/download/).
 
+### Important:
+
+If you're using the latest version of JDK, you'll need to change the Gradle version of your project so it can recognize the JDK.
```

Comment:
It would be nice to use the notice box here (to achieve that you need just add ">" - quotation mark in markdown - at the beginning of the line).